### PR TITLE
Bm test cleanup

### LINKED ISSUE #2: Check Variable Types (let, const)

### LINKED ISSUE #15: Refactor testing to only use data created in test

### LINKED ISSUE #33: Remove unnecessary variables
Use Lint to check project for unnecessary variables

### LINKED ISSUE #37: Remove comments

### LINKED ISSUE #58: Lint code

### PR DESCRIPTION
**Pull request by bretm9**
## Description
User's pantry now stays up to date with API.
Cleaned up tests,
Linted code.

## Context
We linted in order to match Turing's style guides, and cleaned up testing that had broken while refactoring.

## Where To Start
run `npm run lint`

## Affected areas of application
Most files have been edited for linting, but functionality has remained untouched.

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] npm run lint
- [x] npm test
- [x] Chrome browser inspector

## Relevant Tickets
Closes #58,
closes #33,
closes #15,
closes #2,
closes #37

## Tags
N/A

## Questions or Notes (if applicable):
There are a few linting warnings having to do with maximum line length. I attempted to shorten any lines possible.

### CODE REVIEW

**Comment by Chriscastanuela** [comment]
This is very well listed. I have no complaints

File: src/user.js [hunk 1]
```diff
@@ -135,6 +135,17 @@ class User {
...
+        fetch("https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData")
+          .then(response => response.json())
+          .then(data => {
+            let userData = data.wcUsersData.find(user => {
+              return user.id === Number(userId);
+            });
+            this.pantry = userData.pantry;
+          })
+          .catch(err => console.log("err", err))
+      )
```
Comment (holladayian) [inline src/user.js]: Nested fetches are crazy! This definitely makes more sense, now that I'm looking at it.

**Review by holladayian — APPROVED** [review]
I would definitely like to go over some fetch stuff if we get the time.

File: src/scripts.js [hunk 2]
```diff
@@ -21,35 +22,36 @@ window.onload =
...
-  let userId = 1;
-  fetch("https://fe-apps.herokuapp.com/api/v1/whats-cookin/1911/users/wcUsersData")
-  .then(response => response.json())
-  .then(data => {
-  newUser = data.wcUsersData.find(user => {
-    return user.id === Number(userId);
-  });
-  user = new User(userId, newUser.name, newUser.pantry);
-  pantry = new Pantry(newUser.pantry);
+function setUserData() {
```
Comment (holladayian) [inline src/scripts.js:49]: What what the reason for changing the name of this function? Just a curiosity, other than that it looks good!

**Review by holladayian — APPROVED** [review]
Thanks for hooking it up with the fetch! Nice job thinking of using a third param for user.cook

**Pull request merged**